csp-mixin/alerts/azure-alerts.yml

```diff
@@ -22,9 +22,135 @@ groups:
         keep_firing_for: 10m
         labels:
           severity: critical
-          service: 'Azure Virtual Machines.'
+          service: 'Azure Virtual Machines'
           namespace: cloud-provider-azure
         annotations:
           summary: 'VM unavailable.'
           description: 'The VM {{ $labels.resourceName }} is not functioning or crashed, which may require immediate action.'
           dashboard_uid: '58f33c50e66c911b0ad8a25aa438a96e'
+
+      - alert: AzureDatabaseHighDtuConsumption
+        expr: |
+          avg by (job,resourceGroup,subscriptionName,resourceName) (azure_microsoft_sql_servers_databases_dtu_consumption_percent_average_percent{job=~".+",resourceGroup=~".+",subscriptionName=~".+",resourceName=~".+"}) > 90
+        for: 10m
+        keep_firing_for: 10m
+        labels:
+          severity: critical
+          service: 'Azure SQL database'
+          namespace: cloud-provider-azure
+        annotations:
+          summary: 'High database DTU consumption.'
+          description: 'Check active queries and optimize indexes or consider scaling up DTUs to handle load in {{ $labels.resourceName }} database.'
+          dashboard_uid: '82c5b6cf30db5b601c5cc3f5d8d4284d'
+
+      - alert: AzureDatabaseHighStorageUsage
+        expr: |
+          avg by (job,resourceGroup,subscriptionName,resourceName) (azure_microsoft_sql_servers_databases_storage_percent_maximum_percent{job=~".+",resourceGroup=~".+",subscriptionName=~".+",resourceName=~".+"}) > 95
+        for: 15m
+        keep_firing_for: 10m
+        labels:
+          severity: critical
+          service: 'Azure SQL database'
+          namespace: cloud-provider-azure
+        annotations:
+          summary: 'High database Storage usage.'
+          description: 'Archive or delete old data, or scale up storage capacity in {{ $labels.resourceName }} database.'
+          dashboard_uid: '82c5b6cf30db5b601c5cc3f5d8d4284d'
+
+      - alert: AzureDatabaseHighDeadlockCount
+        expr: |
+          sum by (job,resourceGroup,subscriptionName,resourceName) (azure_microsoft_sql_servers_databases_deadlock_total_count{job=~".+",resourceGroup=~".+",subscriptionName=~".+",resourceName=~".+"}) > 5
```

Comment:
I think we need a rate because this is a `count` metric

Reply:
yeah I think you are right. Thanks